config.xml

```diff
@@ -2,7 +2,7 @@
 <module>
 	<name>ps_googleanalytics</name>
 	<displayName><![CDATA[Google Analytics]]></displayName>
-	<version><![CDATA[4.2.2]]></version>
+	<version><![CDATA[4.3.0]]></version>
```

Comment:
If we drop old compatibility, maybe we should do it in a 5.0.0 ? So people understand it's a big change?

Reply:
@Hlavtox it is actually **required** to bump the major version if we change PS compatibility version :)

Reply:
https://github.com/PrestaShop/ADR/blob/master/0020-module-version-bump-after-changing-min-core-version.md